eslint-config-pwn.js

```diff
@@ -123,6 +123,16 @@ module.exports = {
         "@typescript-eslint/explicit-module-boundary-types": "off",
       },
     },
+    {
+      files: ["*.js"],
+      rules: {
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-use-before-define": "off",
+        "@typescript-eslint/no-useless-constructor": "off",
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "@typescript-eslint/no-empty-function": "off",
+      },
+    },
   ],
```

Comment:
i think this doesnt do the trick unfortunately, since there is a lot of more rules applied that comes from this `extends:`

```
    "plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended",
```

probably not extending these rules for `.js` files could do the trick